src/pages/authentication/Login.jsx

```diff
@@ -0,0 +1,46 @@
+import "styles/authentication/Login.css";
+
+const Login = () => (
```

Comment:
if there is only one component you want to export you can directly do `export const Login = () => {}`

Reply:
to main consistency I would export it later